src/components/SelectComponents/Autosuggest/__tests__/Autosuggest.spec.js

```diff
@@ -124,6 +131,16 @@ describe('Autosuggest', () => {
             expect(wrapper.find('input')).toHaveLength(1);
             expect(wrapper.find('.Autosuggest__input--hidden')).toHaveLength(1);
         });
+        it('should show empty suggestion placeholder while results are empty and input has value', () => {
+            wrapper.setProps({ noSuggestionsPlaceholder: 'no suggestions' });
+            setFocusOnInput();
+            expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
I find snapshots are very hard to understand and very easy to mess up if something changes (just update them and the test passes). Could you add some assertions of what in the snapshot are we looking for?